source/tests/system/nidcpower_driver_api_tests.cpp

```diff
@@ -331,6 +343,50 @@ TEST_F(NiDCPowerDriverApiTest, SetAttributeViInt64_GetAttributeViInt64ReturnsSam
   EXPECT_EQ(expected_value, get_attribute_value);
 }
 
+TEST_F(NiDCPowerDriverApiTest, ConfigureOutputFunctionAndVoltageLevel_ConfiguresSuccessfully)
+{
+  const char* channel_name = "0";
+  ViReal64 expected_voltage_level = 3.0;
+  ViInt32 expected_output_function_value = dcpower::OutputFunction::OUTPUT_FUNCTION_NIDCPOWER_VAL_DC_VOLTAGE;
+  configure_output_function(channel_name, expected_output_function_value);
+  ::grpc::ClientContext context;
+  dcpower::ConfigureVoltageLevelRequest request;
+  request.mutable_vi()->set_id(GetSessionId());
+  request.set_channel_name(channel_name);
+  request.set_level(expected_voltage_level);
+  dcpower::ConfigureVoltageLevelResponse response;
+  ::grpc::Status status = GetStub()->ConfigureVoltageLevel(&context, request, &response);
+  EXPECT_TRUE(status.ok());
+  EXPECT_EQ(kdcpowerDriverApiSuccess, response.status());
+
+  ViReal64 actual_voltage_level = get_real64_attribute(channel_name, dcpower::NiDCPowerAttributes::NIDCPOWER_ATTRIBUTE_VOLTAGE_LEVEL);
+  ViInt32 actual_output_function_value = get_int32_attribute(channel_name, dcpower::NiDCPowerAttributes::NIDCPOWER_ATTRIBUTE_OUTPUT_FUNCTION);
+  EXPECT_EQ(expected_voltage_level, actual_voltage_level);
+  EXPECT_EQ(expected_output_function_value, actual_output_function_value);
+}
+
+TEST_F(NiDCPowerDriverApiTest, ConfiguresOutputFunctionAndCurrentLevel_ConfiguresSuccessfully)
```

Comment:
Typo. YOu Configure instead of Configures in Given part of test name

Reply:
Yes, it looks better now.